code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -451,11 +456,17 @@
 /obj/machinery/portable_atmospherics/canister/proc/canister_break()
 	disconnect()
 	var/datum/gas_mixture/expelled_gas = air_contents.remove(air_contents.total_moles())
+	var/expelled_pressure = expelled_gas?.return_pressure()
 	var/turf/T = get_turf(src)
 	T.assume_air(expelled_gas)
 	air_update_turf(FALSE, FALSE)
-
 	obj_break()
+
+	if(expelled_pressure > pressure_limit)
+		var/pressure_dif = expelled_pressure - pressure_limit
+		var/explosion_range = CEILING(min(pressure_dif, 20000) / 1000, 1)
```

Comment:
Can you make these magic numbers defines?

Reply:
Why? They're only relevant here